public/app/plugins/datasource/cloudwatch/components/Forms.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { InputHTMLAttributes, FunctionComponent } from 'react';
+import { Input, FormLabel } from '@grafana/ui';
+
+export interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  label: string;
+  inputEl?: React.ReactNode;
+  tooltip?: string;
+}
+
+export const FormField: FunctionComponent<Partial<Props>> = ({ label, inputEl, className, tooltip, ...inputProps }) => (
+  <>
+    <FormLabel width={8} className="query-keyword" tooltip={tooltip}>
+      {label}
+    </FormLabel>
+
+    {inputEl || <Input type="text" className={`gf-form-input`} {...inputProps} />}
+  </>
+);
+
+export const InlineFormField: FunctionComponent<Props> = ({ ...props }) => {
```

Comment:
If possible these FormField and InlineFormField could maybe be named QueryField and InlineQueryField? So we do no confuse them with general form fields components defined in @grafana/ui

Reply:
yes!